zio/flags.go

```diff
@@ -81,4 +81,6 @@ func (w *WriterFlags) SetFlags(fs *flag.FlagSet) {
 	fs.IntVar(&w.Zng.StreamRecordsMax, "b", 0, "limit for number of records in each ZNG stream (0 for no limit)")
 	fs.IntVar(&w.Zng.LZ4BlockSize, "znglz4blocksize", zngio.DefaultLZ4BlockSize,
 		"LZ4 block size in bytes for ZNG compression (nonpositive to disable)")
+	fs.Float64Var(&w.Zst.ColumnThresh, "coltresh", 5, "minimum frame size (MiB) used for zst columns")
+	fs.Float64Var(&w.Zst.SkewThresh, "skewtresh", 25, "minimum skew size (MiB) used to group zst columns")
```

Comment:
In `zar import` @alfred-landrum introduced `github.com/alecthomas/units` which allows us to do things like this:
```golang
	f.StringVar(&c.thresh, "s", units.Base2Bytes(archive.DefaultLogSizeThreshold).String(), "target size of chunk files, as '10MB' or '4GiB', etc.")
```
I would suggest using this here. Also helps to have consistency with how we specify data size with all our command line tools.

Reply:
👍  to this idea! I'd recently made the dumb mistake of typing `-sortmem 100000` when I thought I'd typed `-sortmem 10000` and all the while of course I was _trying_ to ask for 10 GiB, and hilarity ensued. I doubt I would have made the same mistake if I'd been able to explicitly request 10 GiB.

Reply:
Ok, I'll put up a PR that addresses this (units.Base2Bytes should involved a flag.Var) as well as #1279.  The flags stuff needs another refactor to kill two birds here.

Reply:
See PR #1323

Reply:
Done